### DISCUSSION TITLE
Need help with presentation

### DISCUSSION THREAD

**Discussion by jaanaltosaar** [markdoc/markdoc#340] [Q&A] [answered]
Hi! I started a nonprofit and our websites are all based on Markdoc -- thank you!

However, in reading https://stripe.com/blog/markdoc -- I saw this example deck:

<img width="918" alt="image" src="https://user-images.githubusercontent.com/5317244/224168609-4b8d651d-5537-4798-9a00-a71078568da3.png">

I can't find the code for this. Can you please help? I'm giving a talk at the NYC School of Data next week: https://sched.co/1JUZc

**Comment by rpaul-stripe** [comment, marked as answer]
To make that slide deck, I created a set of Markdoc tags that wrap the [Spectacle.js](https://formidable.com/open-source/spectacle/) React library. I originally wrote that code in 2020, so it is built against an old version of Markdoc and an old version of Spectacle.js, so it doesn't really work out of the box anymore. Both libraries have changed their APIs a fair amount since then. 

That said, I do still have the code, which I've posted for you [here](https://gist.github.com/rpaul-stripe/0c2158c7118e2b8962847d90b5372d25). It won't actually work anymore, but it might be a useful starting point. You would have to update the schema to reflect the current API of Spectacle.js.

**Comment by jaanaltosaar** [comment]
You're the best thank you so much @rpaul-stripe  -- will give that a shot :)) 🙏 🙏 🙏
